bot.py

```diff
@@ -73,6 +75,7 @@ async def on_stopping(event: hikari.StoppingEvent) -> None:
 
 commons.agents.load()
 bot.load_extensions_from("./commands/", must_exist=True)
+bot.load_extensions_from("./message_menu_commands/", must_exist=True)
```

Comment:
Can we move this to `./commands/menu/`. This feels too wordy for a directory to me

Reply:
There are 3 types of commands:
1. Slash Commands
2. Message Menu Commands
3. User Menu Commands

This command is moved under `./commands/menu/message`

I'll add a new command under `./commands/menu/user` which will show meme stats